@navikt/navno/css/card-large.css

```diff
@@ -0,0 +1,51 @@
+.navds-card-large {
+  display: block;
+  margin-bottom: var(--navds-spacing-7);
```

Comment:
Jeg lurer på om `margin-bottom` kan gjøre kortet vanskeligere å ta i bruk i ulike sammenhenger, f.eks som en del av en `display: flex`-gruppe der man vil løse avstander med `gap`. Jeg kjenner ikke til de ulike sammenhengene kortet er tenkt brukt, men er en mulighet å flytte ansvaret for avstander over til parent-elementet?

Reply:
Enig! Fjerner margin.